src/server/auth/server/testing/auth_test.go

```diff
@@ -1595,49 +1334,6 @@ func TestCreatePipelineRepoAlreadyExistsPermissions(t *testing.T) {
 	))
 }
 
-// TestAuthorizedNoneRole tests that Authorized(user, repo, NONE) yields 'true',
-// even for repos with no ACL
-func TestAuthorizedNoneRole(t *testing.T) {
```

Comment:
There's no longer a notion of None scope, so this isn't necessary. 

Also "None scope" makes me think of ![](https://external-preview.redd.it/lApf4dLuIrnlR7xlUilsJNP8u1gjGp30NzV9Gfrr-GA.jpg?auto=webp&s=d0a902d724160e711f97f005bf7125ce22b77bdd)

Reply:
:joy:

Reply:
Literally laughing